### PR TITLE
MCW changes for RALI

### PR DESCRIPTION
**Pull request by LokeshBonta**
- Add new MCW-RALI related Changes
- Modified Crop Mirror Normalize 
- Excluded TF changes 
- Excluded Partial Decoding Chnages

### CODE REVIEW

File: rali/CMakeLists.txt [hunk 1]
```diff
@@ -72,24 +72,29 @@ if(${BUILD_RALI})
...
-			    ./include
-			    )
-	link_directories(${AMDRPP_LIBRARIES_DIR} ${TurboJpeg_LIBRARIES_DIR})
+			    ./include)
+			    #./tf/)
+
+	link_directories(${AMDRPP_LIBRARIES_DIR} ${TurboJpeg_LIBRARIES_DIR}
+					/usr/local/lib/)
 	file(GLOB SOURCES "./source/*.cpp")
+			#"./tf/example.pb.cc"
```
Comment (kiritigowda) [inline rali/CMakeLists.txt:81]: @LokeshBonta any reason all this TF code is commented out?

Comment (LokeshBonta) [reply rali/CMakeLists.txt:81]: @kiritigowda ,
Right now, we are working on some changes suggested by Rajy yesterday for the protobuf issue and we will give TF changes as another PR.

File: rali/README.md [hunk 2]
```diff
@@ -75,6 +82,24 @@ sudo make install
 sudo apt-get install libjsoncpp-dev
 ```
 
+## Google Protobuf installation
```
Comment (rrawther) [inline rali/README.md:85]: To be removed from this readme and add it to setup script

**Comment by LokeshBonta** [comment]
> Please make sure it builds and tests are passing

Hi @rrawther, we have tested with rali_unittests, image-augmentation app and python  test-training.py.
We did not see any issues

File: rali/CMakeLists.txt (hunk 1 above)
Comment (LokeshBonta) [reply rali/CMakeLists.txt:81]: @kiritigowda  We will remove the commented part.

**Comment by rrawther** [comment]
@kiritigowda: Just approved this. Please merge the PR

**Pull request merged**